test/rest/AccessTokenTest.ts

```diff
@@ -0,0 +1,87 @@
+import { AccessToken } from '../../src/rest/AccessToken';
+import { times } from 'underscore';
+
+export function AccessTokenTest() {
+  describe('AccessToken', () => {
+    let token: AccessToken;
+
+    beforeEach(() => {
+      token = new AccessToken('el accesso tokeno');
+    });
+
+    it('should correctly detect an expired status code', () => {
+      expect(token.isExpired({ statusCode: 419 })).toBeTruthy();
+    });
+
+    it('should correctly detect a status code that is not expired', () => {
+      expect(token.isExpired({ statusCode: 500 })).toBeFalsy();
+    });
+
+    it('should behave correctly if there is no renew function', async done => {
```

Comment:
Could the test description be improved? `should behave correctly` does not explain what the expected behaviour is.